cbLazyLoad/dist/cargobay.lazyload.min.css

```diff
@@ -0,0 +1,29 @@
+/* ==========================================================================
+   Toggle
+   ========================================================================== */
+/* Effects
+   ========================================================================== */
+.lazy-load__fade {
```

Comment:
I wouldn't add functionality to include effects, this gives extra bulk which isn't always required. In my opinion cargobay components have to provide the functionality in it's most abstract form, in this case replacing the image. Thoughts?